### PR TITLE
Trunk upgrade

### PR DESCRIPTION
**Pull request by InnocentBug**
# Description
Updating trunk to a new version.

## Known Issues

Something with ruff came up during `./trunk check all` but I don't think that matters.

## Checklist

- [x] My name is on the list of contributors (`CONTRIBUTORS.md`) in the pull request source branch.
- [x] I have updated the documentation to reflect my changes.

### CODE REVIEW

File: docs/images/CRIPT_full_logo_colored_transparent.png
Comment (nh916) [inline docs/images/CRIPT_full_logo_colored_transparent.png:1]: just curious, what changed here?

**Review by nh916 — APPROVED** [review]
Love it!

File: docs/images/CRIPT_full_logo_colored_transparent.png
Comment (InnocentBug) [reply docs/images/CRIPT_full_logo_colored_transparent.png:1]: no idea

**Comment by nh916** [comment]
### be sure to merge this into `develop` and not `main`

**Comment by InnocentBug** [comment]
I don't know why this fails.
Trunk passes, but something doesn't go along with the setting on github.

**Comment by nh916** [comment]
> I don't know why this fails. Trunk passes, but something doesn't go along with the setting on github.

I'm thinking since it is non-essential and it is breaking CI we don't merge it yet, what are your thoughts

**Pull request closed without merging**